src/core.py

```diff
@@ -118,6 +118,13 @@ def __init__(self) -> None:
         self.image_loaded = False
         self.default_font = None
         self.item_tag_dict = {}
+        self.rect_item_tag_dict = {}
+        self.buff_data = None
+        self.img_size = [None,None]
```

Comment:
rename to bright_img_size
all the image sizes of heatmap, rectangle_canvas should be based on bright image size